src/main/java/de/tum/in/www1/artemis/repository/QuizExerciseRepository.java

```diff
@@ -76,11 +75,6 @@ default QuizExercise findOne(Long quizExerciseId) {
      * @param quizExerciseId the id of the entity
      * @return the entity
      */
-    @Nullable
-    default QuizExercise findOneWithQuestions(Long quizExerciseId) {
-        return findWithEagerQuestionsById(quizExerciseId).orElse(null);
-    }
-
     @NotNull
     default QuizExercise findByIdWithQuestionsElseThrow(Long quizExerciseId) {
```

Comment:
Same as above, should change this method to  "findByIdWithQuestionsAndBatchesElseThrow"?